data-table/__tests__/table.js

```diff
@@ -1,63 +1,80 @@
-// test 4-5 function
 const table = require('../table.js');
 
 beforeEach(() => {
     document.body.innerHTML = '<div id="div-id"></div>'
 });
 
-describe('test createDataTable', () => {
-      
-  test('test0', () => {
-    const config = {
-        'wrapperDivId': 'div-id',
-    };
-    table.createDataTable(config);
-    const contents = document.getElementById(config.wrapperDivId).textContent;
-    expect(contents.substr(0, 10)).toEqual("Add column");
-  });
+describe('basic tests to ensure createDataTable can function well', () => {
```

Comment:
For each of these tests, you begin with the same code: creating a default table. Consider using `beforeEach` to avoid copying/pasting this code:
https://jestjs.io/docs/setup-teardown

```
  beforeEach(() => {
        const config = {
            'wrapperDivId': 'div-id',
        };
        table.createDataTable(config); 
});
```